packages/legend-extension-dsl-serializer/src/models/protocols/pure/DSLSerializer_PureProtocolProcessorPlugin.ts

```diff
@@ -135,22 +141,31 @@ export class DSLSerializer_PureProtocolProcessorPlugin
           );
           const element = getBinding(path, context.graph);
           element.schemaId = elementProtocol.schemaId;
-          element.schemaSet = elementProtocol.schemaSet;
-          element.contentType = guaranteeNonEmptyString(
-            elementProtocol.contentType,
-            `Binding 'contentType' field is missing or empty`,
+          if (elementProtocol.schemaSet !== undefined) {
```

Comment:
Instead, why don't you assign `element.schemaSet` to an `OptionalPackageableElementReference` here, if the protocol value is `undefined`, hence set that to the value of the `OptionalPackageableElementReference`.